src/os/FileJournal.cc

```diff
@@ -1242,7 +1249,7 @@ void FileJournal::write_thread_entry()
     }
     
 #ifdef HAVE_LIBAIO
-    if (aio) {
+    if (aio && !g_conf->journal_dynamic_throttle) {
```

Comment:
Disabling the behavior allows us to build up a ton of tiny aio requests.  Why would journal_dynamic_throttle make that ok?